picard/tagger.py

```diff
@@ -208,6 +212,7 @@ class Tagger(QtWidgets.QApplication):
     cluster_removed = QtCore.pyqtSignal(Cluster)
     album_added = QtCore.pyqtSignal(Album)
     album_removed = QtCore.pyqtSignal(Album)
+    translations_updated = QtCore.pyqtSignal()
```

Comment:
This is a bit misleading, as it is really only about the qt translators. I would suggest to be more specific here and probably also mark it with an underscore for internal use and name it `_qt_translators_updated`.